src/helix-ui/styles/core/hx-tile.less

```diff
@@ -0,0 +1,74 @@
+hx-tile {
+  &:extend(.hxBox-lg);
+  border-radius: 2px;
+  display: inline-block;
+  height: 100%;
+  position: relative;
+  text-align: center;
+  width: 100%;
+
+  hx-tile-title {
```

Comment:
* Split out into `core/hx-tile-title.less`
* Leave `margin-bottom` in this file where it is.